app/api/support/DownstreamResponseMappingSupport.scala

```diff
@@ -14,42 +14,44 @@
  * limitations under the License.
  */
 
-package v1.support
+package api.support
 
+import api.controllers.EndpointLogContext
+import api.models
+import api.models.errors
+import api.models.errors._
+import api.models.outcomes.ResponseWrapper
 import utils.Logging
-import v1.controllers.EndpointLogContext
-import v1.models.errors._
-import v1.models.outcomes.ResponseWrapper
 
 trait DownstreamResponseMappingSupport {
   self: Logging =>
 
   final def mapDownstreamErrors[D](errorCodeMap: PartialFunction[String, MtdError])(responseWrapper: ResponseWrapper[DownstreamError])(implicit
-                                                                                                                                       logContext: EndpointLogContext): ErrorWrapper = {
+      logContext: EndpointLogContext): ErrorWrapper = {
 
     lazy val defaultErrorCodeMapping: String => MtdError = { code =>
       logger.warn(s"[${logContext.controllerName}] [${logContext.endpointName}] - No mapping found for error code $code")
-      InternalError
+      errors.InternalError
     }
 
     responseWrapper match {
       case ResponseWrapper(correlationId, DownstreamErrors(error :: Nil)) =>
-        ErrorWrapper(correlationId, errorCodeMap.applyOrElse(error.code, defaultErrorCodeMapping), None)
+        errors.ErrorWrapper(correlationId, errorCodeMap.applyOrElse(error.code, defaultErrorCodeMapping), None)
```

Comment:
can import InternalError & ErrorWrapper so the "errors." and "models.errors." aren't needed